.github/workflows/npm-publish.yml

```diff
@@ -18,6 +18,17 @@ jobs:
       - run: npm ci
       - run: npm test
 
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm link
+      - run: monika -v
```

Comment:
I might be wrong, but I don't think this will catch the errors/crashes of the released package. We need to actually pack the package and install then run the packed version as I mentioned in the issue #73 .